noggin/templates/user-settings-agreements.html

```diff
@@ -0,0 +1,51 @@
+{% extends "user-settings.html" %}
+
+{% import '_form_macros.html' as macros %}
+
+{% block settings_content %}
+<div class="list-group">
+    {% for agreement in agreementslist %}
+    <div class="list-group-item">
+        <div class="row align-items-center">
+          <div class="col h6">
+            <div>{{ agreement.name }}</div>
+          </div>
+          <div class="col-auto">
+            {% if user.username in agreement.users %}
+            {# we use the unique id as the modal ID here, because the name may have spaces in it#}
+            <a data-toggle="modal" data-target="#agreement-modal-{{agreement.uniqueid}}" class="btn btn-link" href="javascript:void(0)">{{ _("View agreement") }}</a>
```

Comment:
`href="javascript:void(0)"` ? Is this really needed?

Reply:
yeah, <a>'s without href's tend to do weird things. the two options i know of are href="#" and javascript:void(0). is there an issue with having this here that you know of?